### DISCUSSION TITLE
"User could not be added" Error Message when Adding to a Repo

### DISCUSSION THREAD

**Discussion by ergodigital** [community/community#64197] [Repositories]
### Select Topic Area

Bug

### Body

I think this is a bug. I have a private repo, but allows collaborators and has no other user-related security switched on. 

We are both based in the UK so clearly it can't be the location. But I click 'Add People', I find the Collaborator, I select his name and then it pauses for a moment and then a red bar is displayed:
"User Could Not be Added" with no further clue as to why not.

I have duplicated it, and tried the same user: Could Not be Added.
I have tried inviting another GitHub account: Could Not be Added.
I created an empty Repository: Could Not be Added.

It feels like a bug... but just hoping that someone may be able to point me in the right direction. It is a Free tier account BUT it does say unlimited collaborators.

Also, FWIW I cannot transfer ownership of the repo to a Collaborator either. It looks locked.

**Comment by ProyectoTuTutorPUJ** [comment]
We (as organization) tried to turn our account as an orgazation, instead of individual one. That configuration resulted in not being able to add a collaborator in any repository, or neither create an organization (because we alredy were one).
We solved by changing the name of my user profile, and then changed it again to the original one.
